lib/ansible/modules/network/iosxr/iosxr_user.py

```diff
@@ -275,6 +357,18 @@ def main():
                            mutually_exclusive=mutually_exclusive,
                            supports_check_mode=True)
 
+    if module.params['publickeyfile']:
+        if not HAS_B64:
+            module.fail_json(
+                msg='library base64 is required but does not appear to be '
```

Comment:
Needs adding to the DOCUMENTATION block
```
requirements:
  - base64 when using I(publickeyfile)
  - paramiko when using I(publickeyfile)
```

http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block

Reply:
Done, although in the newer version it's going to be called public_key_contents.
I'll push again once I've fixed some errors. Added some integration tests as well.